pkg/controller/controller.go

```diff
@@ -100,11 +96,11 @@ func (c *Controller) Run(ctx context.Context) error {
 	c.log.Info("starting workers")
 	// Launch two workers to process Foo resources
```

Comment:
Somewhat related. I noticed this when debugging. This comment does not reflect the constant.

Reply:
Cheers :+1: